Networking/WebSockets/WebSocketProtocol.hh

```diff
@@ -160,18 +160,18 @@ private:
     inline bool consumeMessage(T payLength, char *&src, unsigned int &length, frameFormat frame, void *user) {
         if (getOpCode(frame)) {
             if (opStack == 1 || (!lastFin && getOpCode(frame) < 2)) {
-                forceClose(user);
+                forceClose(user, "cm-1");
```

Comment:
Can we provide the reason along with the some properties of the frame such as reserved bits, opcodes, control frames, compression, and payload length? This info could be helpful to the customer to diagnostic the problems on their side.

Reply:
Also any change to this file should be marked as WebSocketProtocol.hh is from a 3rd party so if we need to update this file, we know what we have made change to the file.

Reply:
Ah, this is subtle: an external header file inside our directory. Let me try to organize it a bit.